nmon/riscv64.patch

```diff
@@ -0,0 +1,12 @@
+--- PKGBUILD
++++ PKGBUILD
+@@ -12,8 +12,7 @@ source=("https://downloads.sourceforge.net/$pkgname/lmon$pkgver.c")
+ sha256sums=('5dc6045f6725e3249a969918fae69663a1d669162087720babcdb90fce9e6b2a')
+ 
+ build() {
+-  cd "$srcdir"
+-  cc -o nmon nmon$pkgver.c $LDFLAGS $CFLAGS -g -O3 -lncurses -lm -D X86
++  cc -o nmon lmon$pkgver.c $LDFLAGS $CFLAGS -g -O3 -lncurses -lm -D X86
```

Comment:
Are you sure this fixes the build on riscv64? You even still have the current architecture defined as x86.

Reply:
> Are you sure this fixes the build on riscv64? You even still have the current architecture defined as x86.

Thank you for your reminder.I'm confident that the test is valid. As a newcomer, I'm still relatively slow at reviewing source code. I understand that 'X86' is a preprocessing macro. What perplexes me is why there was a 404 error during our testing, while in your own testing, the URL was able to download successfully.

Reply:
The point is to port the package to riscv64 architecture, so you need to make sure that:
 + The changes you made is arch-specific
 + The artifacts in the package actually *works* on riscv64